client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -199,7 +198,7 @@ <h3 class="oui-header_4 oui-tile__title" data-translate="domain_dashboard_securi
                                                     data-uib-tooltip="{{(ctrlTabGeneralInformations.domain.protection === 'unlocking' ||
                                                       ctrlTabGeneralInformations.domain.protection === 'locking' ||
                                                       ctrlTabGeneralInformations.domain.protection === 'unavailable') ? ('domain_dashboard_protection_' + ctrlTabGeneralInformations.domain.protection | translate) : ''}}"
-                                                    data-uib-tooltip-append-to-body="true"></span>
+                                                    data-tooltip-append-to-body="true"></span>
```

Comment:
Can't it be replaced by a `oui-tooltip` here ?

Reply:
Hello, it generates issues with switch, not planned in the scope of this PR. 
A next iteration is planned 😄